cmd/liqonet/network-manager.go

```diff
@@ -55,7 +55,8 @@ func addNetworkManagerFlags(managerFlags *networkManagerFlags) {
 }
 
 func validateNetworkManagerFlags(managerFlags *networkManagerFlags) error {
-	cidrRegex := regexp.MustCompile(`^(\d{1,3}.){3}\d{1,3}(/(\d|[12]\d|3[012]))$`)
+	cidrRegex := regexp.MustCompile(`^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]` +
+		`|1[0-9]{2}|2[0-4][0-9]|25[0-5])(\/(3[0-2]|[1-2][0-9]|[0-9]))$`)
```

Comment:
IIRC, @aleoli wrote a custom argument parser for network CIDRs. Don't know whether it might be worth to switch using that one instead of the regex.

Reply:
Do you have a pointer to the function?

Reply:
Yes, that struct implements the flag.Value interface https://github.com/liqotech/liqo/blob/master/pkg/utils/args/cidrlist.go#L23